services/rest_gate/.env

```diff
@@ -0,0 +1 @@
+../../.env
```

Comment:
that is how GH shows symlinks?

Reply:
Yes https://github.com/nspcc-dev/neofs-dev-env/blob/446bbb6a564ec6ccedfe6f4ed3bb2509511257b0/services/storage/.env